addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1287,7 +1287,47 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
      */
     _computeWidgetVisibility: function (widgetName, params) {
         const dependencyEl = this._getDependencyEl();
+        const currentEl = this._getCurrentEl();
```

Comment:
`_getCurrentEl()` function is really needed? used only once, we can move that code to here.

`this.el.querySelector(".s_website_form_input")`

Reply:
i removed it. i created with the thought that it will be required often.